docs/tutorials/quickstart.md

```diff
@@ -31,6 +39,12 @@ The worker can also be started using a custom config file:
     blueapi --config path/to/file serve
 ```
 
+An example of a config file that starts STOMP with default values can be found in:
+
+```
+    src/script/stomp_config.yml
+```
+
```

Comment:
Should: As above with ActiveMQ vs RabbitMQ, if we're going to make rabbitMQ the default for new users then maybe this should be the default config?

Reply:
It used to be, but I changed it. I think it's important for a complete beginner to clone -> build -> run with no extra steps, because once you add 1 extra step it's very easy to get into a "just one more..." mentality. The message bus is an optional feature that adds rich feedback and that we use in production, but configuring it is part 2 of the tutorial.